### DISCUSSION TITLE
Trouble running TakeShape App for the first time

### DISCUSSION THREAD

**Discussion by severinschmid** [vercel/next.js#11531] [Help]
Hey, 
I am relatively new to web development and wanted to start a site with zeit.co and TakeShape. I followed the documentation until the end and thought I did everything correctly, but I can't get the app to run even locally. This is the error message I get: 
"Error serializing `.allPosts` returned from `getStaticProps` in "/". Reason: `undefined` cannot be serialized as JSON. Please use `null` or omit this value all together."
I can't figure out what that means because I don't yet understand the concept of staticProps. Any help would be much appreciated!

**Comment by chibicode** [comment]
@severinschmid Hi there, I wrote the TakeShape example. Sorry that the error message is confusing. I believe it's happening because your takeshape API is returning empty results.

Can you try debugging by adding `console.log(json.data)` before this line on `lib/api.js`?: https://github.com/zeit/next.js/blob/d61eced9a6198d7717796fab873d8596485d459e/examples/cms-takeshape/lib/api.js#L24

**Reply by severinschmid** [reply]
Hi, thanks for the help. This is the json object: "{ message: 'Insufficient permissions', statusCode: 403, error: 'Forbidden' }". I thought this has to do with the permissions of the API Key I generated (with 'Read' permissions). I created another one with Dev permissions, and updated my now secrets and .env, but I still get the same response.
